.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: go
+go:
```

Comment:
Are these the versions we want here given we are claiming support for more versions? It sounds like Travis supports many more versions.